Cargo.toml

```diff
@@ -332,7 +332,7 @@ test-fuzz = "3.0.4"
 thiserror = "1.0.40"
 # tokio is defined at a specific version to support patching with iota-sim
 # Do not upgrade without updating https://github.com/iotaledger/iota-sim first
-tokio = "=1.39.2"
+tokio = "=1.43.0"
```

Comment:
Should we also bump tokio for the examples? I know msim is not involved but maybe to avoid some potential behaviour difference

Reply:
good idea, done